docs/notebooks/DESC_Solve_from_pyQSC.ipynb

```diff
@@ -0,0 +1,250 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2a3fc4c6-14e1-4b42-b95c-ccd69e637c72",
+   "metadata": {},
+   "source": [
+    "# Creating a DESC Equilibrium from a pyQsc Near-Axis Equilibrium"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a6ee32b-1a4c-47f5-9a9d-3498fb7d3f55",
+   "metadata": {},
+   "source": [
+    "Note that you must have pyQsc installed in order to make use of the `Equilibrium.from_near_axis` method, do so with `pip install qsc`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "a1f1791e-97bf-4a56-8968-4650d48f003d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "DESC version 0.7.2+319.g66640f23.dirty,using JAX backend, jax version=0.2.25, jaxlib version=0.1.76, dtype=float64\n",
+      "Using device: CPU, with 21.62 GB available memory\n"
+     ]
+    }
+   ],
+   "source": [
+    "# must have installed pyQsc with `pip install qsc` in order to use this!\n",
+    "from qsc import Qsc\n",
+    "from desc.equilibrium import Equilibrium\n",
+    "from desc.objectives import get_fixed_boundary_constraints\n",
+    "from desc.plotting import plot_comparison"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "90a66694-ccee-43e5-909c-927681091871",
+   "metadata": {},
+   "source": [
+    "DESC is able to create an equilibrium based off of a `pyQsc` NAE equilibrium object. First, we'll make the NAE equilibrium using `pyQsc`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "0403611d-9bd7-44cf-88ff-9c848ba3c1d2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "qsc_eq = Qsc.from_paper(\"precise QA\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed61eac3-54f8-49a2-bb46-23e73e07ea06",
+   "metadata": {},
+   "source": [
+    "Then, to make the DESC equilibrium, the `Equilibrium` class has a method `Equilibrium.from_near_axis`. This method creates a DESC `Equilibrium` based off of the `pyQsc` equilibrium. It requires as input the desired DESC FourierZernike resolution, as well as the radius at which you want to evaluate the qsc equilibrium at to make the DESC equilibrium's boundary. The equilibrium's initial `R_lmn`, `Z_lmn` Fourier-Zernike coefficients are fit to the `R,Z` evaluated from the `pyQsc` equilibrium, and the initial `L_lmn` are 0 (because the `pyQsc` equilibrium uses Boozer angles, so there is no poloidal stream function)"
```

Comment:
Clarification because it's been a while since I implemented this: the poloidal angle is kept as the Boozer poloidal angle, but the toroidal angle is converted to the standard toroidal angle. $(\theta_B, \phi)$ are generally not straight field-line coordinates, so are we sure it's correct to set $\lambda=0$?